aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -745,27 +892,23 @@ module aptos_framework::account {
 
         let account = borrow_global_mut<Account>(resource_addr);
         account.signer_capability_offer.for = option::some(resource_addr);
-        let signer_cap = SignerCapability { account: resource_addr };
+        let signer_cap = SignerCapability {account: resource_addr};
         (resource, signer_cap)
     }
 
     /// create the account for system reserved addresses
-    public(friend) fun create_framework_reserved_account(addr: address): (signer, SignerCapability) {
+    public(friend) fun create_framework_reserved_account(addr: address)
+        : (signer, SignerCapability) {
         assert!(
-            addr == @0x1 ||
-                addr == @0x2 ||
-                addr == @0x3 ||
-                addr == @0x4 ||
-                addr == @0x5 ||
-                addr == @0x6 ||
-                addr == @0x7 ||
-                addr == @0x8 ||
-                addr == @0x9 ||
-                addr == @0xa,
-            error::permission_denied(ENO_VALID_FRAMEWORK_RESERVED_ADDRESS),
+            addr == @0x1 || addr == @0x2 || addr == @0x3 || addr == @0x4 || addr == @0x5
```

Comment:
not sure we could do better here?

Reply:
fixed at https://github.com/movebit/movefmt/commit/06f2daf949649e1b4f9f6b1fb1df64ab7620a26e